style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a:link {
+  color: white;
+}
+
+a:hover {
+  color: red;
+}
+
+/* Tool bar container */
+
+.toolbar {
+  width: 100%;
+  height: 92px;
+  position: fixed;
+  background-color: #3c3a39;
+  left: 0;
+  top: 0;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+/* section container */
+
+.section {
+  background-image: url(img/Group.png);
+  background-repeat: no-repeat;
+  background-position: 292px 60%;
+  background-color: #1c1a19;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 100%;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- kindly make the background image more visible as shown in the Figma design I suggest you, adjust a bit the background-position eg. `background-position: 230px 70%;` it will make your design stick more to the Figma design :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/235544625-90a708f3-b850-4c11-9286-6982cd6400c7.png)


## Figma design 

![image](https://user-images.githubusercontent.com/95297251/235544511-fad1a425-32aa-4b7a-b6a3-048e435413db.png)